faridafaqiri/Roman-numeral-converter
# Roman Numeral Converter

## Overview
This pull request adds a complete Roman Numeral Converter web application. The application allows users to enter a number between 1 and 3999 and convert it to its corresponding Roman numeral. The project includes HTML, CSS, and JavaScript files, along with a responsive design for improved usability on various devices.

## Files Added
1- index.html: Contains the structure of the web application, including an input field, a button, and an output area.
2- script.js: Implements the logic for input validation and conversion of numbers to Roman numerals.
3- styles.css: Provides styling for the application, including layout, typography, and responsive design.

## How to Test
1- Clone the repository to your local machine.
    git clone git@github.com:faridafaqiri/roman-numeral-converter.git
2-Navigate to the project directory.
   cd roman-numeral-converter
3- Open index.html in your web browser.
4- Enter a number between 1 and 3999 in the input field.
5- Click the "Convert to Roman Numeral" button.
6- Verify that the correct Roman numeral is displayed.
7- Test with various inputs, including invalid numbers (e.g., non-numeric values, numbers less than 1 or greater than 3999) to ensure proper error handling.
